.zenodo.json

```
@@ -10,46 +10,46 @@
             "name": "Thomas, E.G.",
             "orcid": "0000-0001-8036-8793"
         },
+        {
+            "affiliation": "Naval Research Laboratory",
+            "name": "Burrell, A.G.",
+            "orcid": "0000-0001-8875-9326"
+        },
         {
             "affiliation": "University of Saskatchewan",
-            "name": "Reimer, A.S.",
-            "orcid": "0000-0002-4621-3453"
+            "name": "Ponomarenko, P.V.",
+            "orcid": "0000-0001-8407-0193"
         },
         {
             "affiliation": "The University Centre in Svalbard",
             "name": "Bland, E.C.",
             "orcid": "0000-0002-0252-0400"
         },
         {
-            "affiliation": "Naval Research Laboratory",
-            "name": "Burrell, A.G.",
-            "orcid": "0000-0001-8875-9326"
+            "affiliation": "Virginia Tech",
+            "name": "Sterne, K.T.",
+            "orcid": "0000-0002-1076-0009"
         },
         {
-            "affiliation": "Lancaster University",
-            "name": "Grocott, A.",
-            "orcid": "0000-0002-6489-095X"
+            "affiliation": "University of Saskatchewan",
+            "name": "Rohel, R.A.",
+            "orcid": "0000-0003-2208-1553"
+        },
+        {
+            "affiliation": "Dartmouth College",
+            "name": Shepherd, S.G.",
```

```suggestion
            "name": "Shepherd, S.G.",
```

there still seems to be a formatting error here

---

Good catch, I couldn't figure out why github was highlighting it strangely!  Sometimes the obvious things blow past me.